### ISSUE TITLE
Make Prozac mitigate Bad Feelings.

### ISSUE THREAD

**Issue by slawr34** [CleverRaven/Cataclysm-DDA#11495]
I was actually surprised the first time I took Prozac it didn't actually affect your morale/counteract bad feelings.
##

**Comment by KrazyTheFox** [comment]
Prozac and other antidepressants don't work that way. It's designed to treat real medical conditions such as anxiety and depression, not temporary bad moods. They also work over long periods of time, sometimes taking weeks to start working. Finally,  antidepressants routinely have brutal side effects that can be far worse than the mental conditions themselves. (I take antidepressants and can't take even small doses of Prozac; I get heart palpitations and it sends my anxiety through the roof.)

A person without mental illness who takes antidepressants will feel no benefit and can suffer many of the same side effects, causing way more harm than good.

TL;DR: Works as intended.

**Comment by Coolthulhu** [comment]
Pretty sure it simply cuts divides negative morale by some number.
The number next to `Bad feeling` will not change, but the sum of all morale effects will be higher than if you didn't take Prozac.

**Comment by KA101** [comment]
I'm inclined to take KrazyTheFox's report on this one.  Any other experiences?

**Comment by Rivet-the-Zombie** [comment]
I've never taken the stuff before, but I'm friends with a couple people who have been prescribed fluoxetine in the past, and it made all of them jittery and kind of hyperactive; one had to be taken back off of it because it increased her heart rate to the point where it was dangerous.

**Comment by DavidKeaton** [comment]
I agree with Fox, I take Lexapro and it does indeed take a while. SRIs need time to build up, such side effects are: headaches, nausea, restlessness, odd dreams, "daydreaming" (you stare off into space often), and oddly enough, suicidal thoughts. it isn't a pop and cure all, it has no effect for those with standard seratonin levels, and will p much just be side effects, as stated.

**Comment by DavidKeaton** [comment]
also on the subject of drugs, Dextroamphetamines last a while, too short in game. :-)

**Comment by illi-kun** [comment]
### This issue was closed as it appears inactive.

Reducing open issues to those which are (or will) be actively worked upon helps us focus our efforts. This issue has not been deleted - it still appears in searches and if it contains relevant information you are encouraged to continue to link to it.

### If this issue was a bug
It should be reopened if it can be reproduced in the current build. You can obtain the most recent copy [here](http://dev.narc.ro/cataclysm/jenkins-latest/). Please check there is not a more recent report of this bug before doing so. If no more recent report exists you should continue the discussion in this issue.

### If this was a feature request
If the consensus was that the idea was good you could consider submitting an implementation via a PR. If you want to comment further please do so here as opposed to opening a new issue. Before posting check nobody has already made the same point and consider whether your comments are likely to lead to an implementation. If you have doubts about either consider instead [voting](http://iv.cataclysmdda.com:8111/all) for the issue

### If you want to work on this issue
Then either assign it to yourself or if you are unable to do so claim it via adding a comment. Please don't assign others or make a general request for action.
